src/vmm/src/devices/virtio/queue.rs

```diff
@@ -886,51 +886,11 @@ mod verification {
     #[kani::proof]
     #[kani::unwind(2)]
     fn verify_set_avail_event() {
-        let ProofContext(mut queue, mem) = kani::any();
+        let ProofContext(mut queue, mem) = ProofContext::bounded_queue();
 
         queue.set_avail_event(kani::any(), &mem);
     }
 
-    #[kani::proof]
-    #[kani::unwind(2)]
-    #[kani::solver(cadical)]
-    fn verify_add_used() {
```

Comment:
May I ask why this needs to be removed instead of just be commented out temporarily?

Reply:
I suppose its just personal preference to not have commented out code in version control. I logged us a ticket to readd it once the kani issue is resolved

Reply:
Then the next commit can be a `git revert`.

Reply:
Got it. Thanks :)